exercise_Shelhon.Rmd

```diff
@@ -0,0 +1,87 @@
+
+7.  Provide a title, short description/notes of your changes, and most importantly under 'Reviewers' on the right, type in the username of your **assigned PR reviewer** (which you can find in the 'PR_reviewer_assignments.md' document in this repository. When you are all set, submit the pull request.
+
+8.  **Review your assigned pull request**. You should receive a notification either via GitHub (you can check by clicking the 'notifications' icon directly next to your profile photo in the upper right corner), or you will receive an email link to the email you registered your GitHub account to.
+
+    Follow [these guidelines for reviewing a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request), give some meaningful feedback, and when you're done 'Approve' the pull request (do NOT close or merge).
+
+## The coding exercise:
+
+Work through the steps below to complete this exercise (to the best of your ability) and then save, add, commit and push this file with your coding edits to your forked repo and follow the instructions above for completing the assignment via creating and reviewing a pull request.
+
+Load in the tidyverse (only package(s) needed for this exercise:
+
+```{r}
+library(tidyverse)
+```
+
+Read in the 'kenya_county_climates.csv' in the 'data/' folder and assign it to an environmental variable (Hint: use 'read_csv()').
+
+```{r}
+library(readr)
+kenya_county_climates <- read_csv("~/test-git-intro/Week-1-Intro/data/kenya_county_climates.csv")
+View(kenya_county_climates)
+```
+
+Write a chain of operations that does the following:
+
+1.  `select()` just the columns that have "Precipitation" in the name (so we are only interested in the precipitation data), AND the 'County' column.
+
+2.  `filter()` all the rows where 'Precipitation_585_90' is **greater than** the mean of all county precipitation values.
+
+3.  `pivot_longer()` all the precipitation columns (so everything *except* 'County'), so we have one column for all precip values and one column with the value's associated variables.
+
+```{r}
+assign_kenya <-kenya_county_climates %>%
+  select(matches("Precipitation|County"))%>%
+  filter(Precipitation_585_90 > mean(Precipitation_585_90)) %>%
+  pivot_longer(cols = -("County"))
+view(assign_kenya)
+  
+```
+
+Question: How many counties have predicted precip values greater than the total average for all counties? ##\*n_distinct(assign_kenya\$County) = 22 distinct counties greater than the mean precip\*
+
+Using the new dataset created above, create a bar plot of precipitation for each time frame for each county using `facet_wrap()`
+
+```{r}
+ggplot(assign_kenya, aes(fill = County)) +
+  geom_col(x = County, y = value)
+
```

Comment:
data_filtered %>% 
  ggplot(aes(x = name, y = value)) +
  geom_col() +
  theme(axis.text.x = element_text(angle = 90)) +
  facet_wrap(~County)
I also did fill = County, good move. I just copied the key here for comparison